Tavisca.Bootcamp.LanguageBasics.Exercise1/Program.cs

```diff
@@ -14,16 +14,84 @@ static void Main(string[] args)
             Console.ReadKey(true);
         }
 
-        private static void Test(string args, int expected)
-        {
+        private static void Test(string args, int expected)
+        {
             var result = FindDigit(args).Equals(expected) ? "PASS" : "FAIL";
-            Console.WriteLine($"{args} : {result}");
+            Console.WriteLine($"{args} : {result}");
         }
 
-        public static int FindDigit(string equation)
-        {
+        public static int FindDigit(string equation)
+        {
             // Add your code here.
-            throw new NotImplementedException();
+            
+            char []equal = {'='};
+            string []arr=equation.Split(equal);

```

Comment:
Variable names should tell what they are, and should not be arr, right, a, b, c etc.
Here the variable name should be "expressions", as when you split an equation you get two expressions.